dms/models/dms_file.py

```diff
@@ -109,10 +109,17 @@ class File(models.Model):
         string="Content",
         attachment=False,
         prefetch=False,
-        required=True,
         store=False,
     )
 
+    attachment_type = fields.Selection(
```

Comment:
```suggestion
    type = fields.Selection(
```
IMO `type` is a better field name.